### ISSUE TITLE
🦠 Model Request: Chemprop Acinetobacter baumannii

### ISSUE THREAD

**Issue by miquelduranfrigola** [ersilia-os/ersilia#785]
### Model Name

Acinetobacter activity prediction

### Model Description

This model is a Chemprop neural network trained with a growth inhibition dataset. Authors screened ~7,500 molecules for those that inhibited the growth of A. baumannii in vitro. They discovered abaucin, an antibacterial compound with narrow-spectrum activity against A. baumannii.

### Slug

chemprop-abaumannii

### Tag

A.baumannii

### Publication

https://www.nature.com/articles/s41589-023-01349-8

### Source Code

https://github.com/GaryLiu152/chemprop_abaucin

### License

GPLv3

**Comment by carcablop** [comment]
Hello @miquelduranfrigola and @GemmaTuron 
I hope you are very well.
I have managed to solve many of the problems I had in my development environment, I had problems with conda, and I had errors in the discrepancies of the installed python versions, the environments I had created were not working in isolation.
Once again I managed to run ersilia without problems and fetch models.
The model is now ready to be tested.

https://github.com/ersilia-os/eos3804/pull/1

Thanks you so much.
